astro.config.mjs

```diff
@@ -0,0 +1,14 @@
+import { defineConfig } from 'astro/config';
+
+import tailwind from "@astrojs/tailwind";
+
+// https://astro.build/config
+export default defineConfig({
+  site: 'https://dreamworksanimation.github.io',
```

Comment:
Should this be updated to aswf?

Reply:
Cool

Reply:
After talking with Matthew I will be updating this to the new site under aswf